include/kllvm/codegen/ProofEvent.h

```diff
@@ -119,6 +120,10 @@ class ProofEvent {
   [[nodiscard]] llvm::BasicBlock *
   functionEvent_post(llvm::BasicBlock *current_block);
 
+  [[nodiscard]] llvm::BasicBlock *scEvent(
```

Comment:
I have a mild preference for:
```suggestion
  [[nodiscard]] llvm::BasicBlock *sideConditionEvent(
```